docs/developer-docs/setup/quickstart.md

```diff
@@ -0,0 +1,167 @@
+# Developer quick start
+
+## Overview
+
+This guide is designed as a minimal quick start tutorial for creating a project on the Internet Computer. Other developer documentation pages provide additional supporting context to the concepts and workflows used within this page, and they will be linked for reference accordingly. 
+
+## Prerequisites
+
+To follow this quick start guide, you will need to have the following:
+
+- A connection to the internet.
+- A command line interface.
+- [Node.js](https://nodejs.org/en) downloaded and installed.
+
+## Install the IC SDK
+
+The IC SDK includes `dfx`, which is a CLI tool used to create projects on the IC. It can be downloaded with the command:
+
+```
+sh -ci "$(curl -fsSL https://internetcomputer.org/install.sh)"
+```
+
+For more information on installing the IC SDK, please see [here](./install/index.mdx)
+
+## Start the local replica
+
+To start a local instance of the IC replica, use the command:
+
+```
+dfx start --background
+```
+
+The `--background` flag will start the replica in the background.
+
+## Create and deploy the template project
+
+To create a new project with `dfx`, run the command:
+
+```
+dfx new project_name
+```
+
+This project is a simple 'Hello, world!' project that includes a simple frontend UI. You can see a more detailed look into this project in the guide [here](/docs/current/developer-docs/setup/deploy-locally#test-the-dapp-frontend).
+
+Then, to create, build, and deploy the project, use the command:
+
+```
+dfx deploy
+```
+
+This command will deploy the project **locally**. To learn more about local development, including how to interact with your canister once deployed, please see [here.](deploy-locally.md)
```

Comment:
```suggestion
This command will deploy the project **locally**. To learn more about local development, including how to interact with your canister once deployed, please see [here](deploy-locally.md).
```
I _think_ the dot belongs after the link, but it's not specified in the style guide

Reply:
Since either placement will result in the same visual formatting on the webpage, they can go in either location.